cmake/modules/libsinsp.cmake

```
@@ -17,13 +17,15 @@ include(ExternalProject)
 include(libscap)
 if(NOT WIN32)
 	include(tbb)
+endif()
+if(NOT WIN32 AND NOT APPLE)
 	include(b64)
 	include(jq)
+	include(curl)
```

```suggestion
	if(NOT MINIMAL_BUILD)
		include(curl)
	endif()
```
We need to exclude `curl` from `MINIMAL_BUILD`. This change should fix the failing test.

---

Can we not add nested ifs? A separate block with `if(NOT WIN32 AND NOT APPLE AND NOT MINIMAL_BUILD)` should be clearer I think

---

addressed in 331670dd65953dd000a80cbb0d0e9e41a0728d52